openshift/moldavite-jupyterbook-build.yaml

```diff
@@ -164,15 +164,33 @@ objects:
 
     - name: await-build
       resubmitPendingPods: true
-      resource:
-        action: get
-        successCondition: status.Conditions.Reason == Succeeded
-        failureCondition: status.Conditions.Reason == Failed
-        manifest: |
-          apiVersion: tekton.dev/v1beta1
-          kind: PipelineRun
-          metadata:
-            name: ${MOLDAVITE_BOOK_ID}
+      serviceAccountName: argo
+      script:
+        image: kubectl
+        imagePullPolicy: Always
+        command: [bash]
+        source: |-
+          set -x
+          while true
+          do
+            state=$(
+              kubectl get pipelineruns -o go-template \
+              --template '{{range .status.conditions}}{{.reason}}{{" "}}{{"\n"}}{{end}}' \
+              "${MOLDAVITE_BOOK_ID}" \
+            )
+            [[ $? -eq 0 ]] || exit 1
```

Comment:
would this true, even if the condition is True, as exit would be 0, on get the output itself.

Reply:
thanks for explanation.(my eyes noticed || as & :sweat: :see_no_evil: )